recipes/anaquin/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -eu -o pipefail
+
+# download and copy required dependencies
+# klib
+wget https://github.com/attractivechaos/klib/archive/spawn-final.tar.gz
+tar xzf spawn-final.tar.gz
+cp -r $SRC_DIR/klib-spawn-final $PREFIX/lib/klib/
+
+# vcflib
+wget https://github.com/student-t/vcflib-forAnaquin/archive/v1.0.0-rc1.tar.gz
+tar xzf v1.0.0-rc1.tar.gz
```

Comment:
@student-t, You could split each 'wget' call into a separate recipe, and just add them to the build/run deps.